components/EventList.js

```diff
@@ -15,6 +16,37 @@ export default class EventList extends React.Component {
         await this.props.reload();
     }
 
+    scrollToNow = () => {
+        let currentHour = new Date();
+        currentHour.setMinutes(0, 0, 0);
```

Comment:
Wait, setMinutes takes *3* arguments?? Who knew?

Reply:
Yeah, it also sets seconds and ms.